linera-service/src/cli/command.rs

```diff
@@ -38,9 +38,14 @@ const DEFAULT_BPS: usize = 10;
 #[derive(Clone, Serialize, clap::Args)]
 #[serde(rename_all = "kebab-case")]
 pub struct BenchmarkCommand {
-    /// How many chains to use for the benchmark
-    #[arg(long, default_value_t = DEFAULT_NUM_CHAINS)]
-    pub num_chains: usize,
+    /// How many chains to use in each of the chain groups. One tokio task will be created per chain
+    /// group, for sending block proposals.
+    #[arg(long, default_value_t = DEFAULT_NUM_CHAINS_PER_CHAIN_GROUP)]
+    pub num_chains_per_chain_group: usize,
+
+    /// How many chain groups to use. If not provided, the number of CPUs will be used.
```

Comment:
(But each chain group task will spend a lot of time waiting for validators' responses, won't it? I guess we'll usually specify this and not use the default anyway…)